blocks/shared-styles/_children/button/index.jsx

```diff
@@ -260,7 +276,7 @@ Button.propTypes = {
 
```

Comment:
The `iconType` prop types should be a `PropTypes.oneOf(['user', 'chevron-up', 'chevron-down'])`

Reply:
updated!